janusgraph-test/src/test/java/org/janusgraph/util/StringEncodingTest.java

```diff
@@ -45,10 +46,15 @@ public void testAsciiStringEncoding() {
             try {
                 StringEncoding.getAsciiByteLength(s);
                 fail();
-            } catch (IllegalArgumentException e) {
-
-            } catch (NullPointerException e) {
-
+            } catch (IllegalArgumentException | NullPointerException e) { }
+
+            if (s != null) {
```

Comment:
I understand that you're cleaning up Coverity findings here, but there's a cleaner way of writing tests that expect to fail, with a particular exception, e.g.,

* https://stackoverflow.com/a/156528
* https://stackoverflow.com/a/41031738
* more details on both: https://blog.goyello.com/2015/10/01/different-ways-of-testing-exceptions-in-java-and-junit/

My recommendation here would be to factor out the tests that are expected to fail into separate test methods, one for each invalid input (since each one fails in a different way). Since this is technically a Coverity cleanup PR, and not a test cleanup PR, it's up to you whether you want to fix it now, or add a TODO comment / file an issue to clean it up as a follow-up change.

Reply:
@mbrukman Pushed fix to clean this up